packages/editor/src/components/post-taxonomies/flat-term-selector.js

```diff
@@ -165,6 +165,8 @@ export function FlatTermSelector( { slug } ) {
 		editPost( { [ taxonomy.rest_base ]: newTermIds } );
 	}
 
+	const { createErrorNotice } = wp.data.dispatch('core/notices'); // Importing dispatch function
```

Comment:
It's better to avoid using global dispatch when writing code for the core or anytime the build process is available for the project. Use hooks instead:

```
const { createErrorNotice } = useDispatch( noticesStore );
```

Group it with other `useDispatch` calls above and you can drop the comment.